/**:
  ros__parameters:
    lane_number: 3
    primitives: [
                6039,	8012,	6025,
                5832,	8018,	1682,
                6117,	8023,	6138,
                5149,	8024,	2322,
                6088,	8025,	6102,
                5326,	8026,	3482,
                5430,	8027,	7,
                6165,	8028,	6179,
                6204,	8029,	6218,
                5693,	8167,	1142,

                6039,	8012,	6025,
                5832,	8018,	1682,
                6117,	8023,	6138,
                ]
    
    pit_stop_primitives: [8002, 8019, 8000,
                          8011, 8010, 8009,]

    goal_pose:
      position:
        x: [
            45193.02815,  45194.12755,  45195.182,
            44956.8137,   44959.25915,  44961.59315,
            45130.91055,  45128.3369,   45125.75465,
            45186.76745,  45184.14915,  45181.53085,
            45854.45265,  45851.78045,  45849.128,
            45955.58875,  45952.92,     45950.2525,
            45915.5132,   45919.00205,  45922.53805,
            45914.35565,  45917.83325,  45921.3732,
            45591.92835,  45594.5994,   45597.2364,
            45405.89765,  45408.6405,   45411.2437,
            45405.89765,  45408.6405,   45411.2437,
            45193.02815,  45194.12755,  45195.182,
            44956.8137,   44959.25915,  44961.59315,
            45130.91055,  45128.3369,   45125.75465,
          ]
        y: [
            45771.2614,   45774.74815,  45778.1358,
            45858.3901,   45861.07885,  45863.70325,
            45973.78155,  45971.4117,   45969.04685,
            45912.31385,  45909.9961,   45907.6735,
            45174.50205,  45172.14585,  45169.7651,
            45062.8031,   45060.3917,   45057.99255,
            44881.8817,   44881.85735,  44881.83925,
            44980.8065,   44980.86095,  44980.9021,
            45433.45105,  45435.7927,   45438.13245,
            45638.76175,  45641.2157,   45643.59205,
            45638.76175,  45641.2157,   45643.59205,
            45771.2614,   45774.74815,  45778.1358,
            45858.3901,   45861.07885,  45863.70325,
            45973.78155,  45971.4117,   45969.04685,
          ]
        z: 0.0
      orientation:
        x: 0.0
        y: 0.0
        z: [
          0.9149307,
          0.987987,
          0.938047,
          -0.403409,
          -0.403908,
          -0.3988,
          -0.40437,
          0.724397,
          0.712491,
          0.912545,
          0.9149307,
          0.987987,
          0.938047,
        ]
        w: [
          0.70168,
          0.154534,
          0.346507,
          0.915019,
          0.92395,
          0.917037,
          0.914799,
          0.689382,
          0.680907,
          0.408974,
          0.70168,
          0.154534,
          0.346507,
        ]